packages/core/src/applicationcomposer/composerWebview.ts

```diff
@@ -5,37 +5,43 @@
 
 import { join } from 'path'
 import * as nls from 'vscode-nls'
-const localize = nls.loadMessageBundle()
 import * as path from 'path'
 import * as vscode from 'vscode'
 import { handleMessage } from './handleMessage'
 import { FileWatchInfo } from './types'
 
+const localize = nls.loadMessageBundle()
+
 export class ApplicationComposer {
     public readonly documentUri: vscode.Uri
-    public webviewPanel: vscode.WebviewPanel
+    public webviewPanel: vscode.WebviewPanel = {} as vscode.WebviewPanel
```

Comment:
is there any reason this can't be `public webviewPanel: vscode.WebviewPanel | undefined`? If the webview panel is an empty object does that make some assumptions down the line that it might be defined, even though it isn't?

Reply:
Yeah, I wasn't sure if there was a better way to handle this. Since the constructor is private, the only way to instantiate `ApplicationComposer` is through the `create` method. Thus, this field will always be initiated. Perhaps there is a better way to enforce this invariant? 

The advantage of an empty object is that I don't have to do an undefined check on each usage.